docs/configuration-parameters.md

```diff
@@ -132,7 +132,7 @@ This section provides the details of all the configurations required for CarbonD
 | Parameter | Default Value | Description |
 |---------------------------------------|---------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | high.cardinality.identify.enable | true | If the parameter is true, the high cardinality columns of the dictionary code are automatically recognized and these columns will not be used as global dictionary encoding. If the parameter is false, all dictionary encoding columns are used as dictionary encoding. The high cardinality column must meet the following requirements: value of cardinality > configured value of high.cardinalityEqually, the value of cardinality is higher than the threshold.value of cardinality/ row number x 100 > configured value of high.cardinality.row.count.percentageEqually, the ratio of the cardinality value to data row number is higher than the configured percentage. |
-| high.cardinality.threshold | 1000000 | Threshold to identify whether high cardinality column.Configuration value formula: Value of cardinality > configured value of high.cardinality. The minimum value is 10000. |
+| high.cardinality.threshold | 1000000 | Threshold to identify whether high cardinality column.if columns of cardinality > the configured value, then the columns don't do dictionary encoding. |
```

Comment:
replace 
| high.cardinality.threshold | 1000000 | It is a threshold to identify high cardinality of the column. If the value of column cardinality > the configured value, then the column is excluded from dictionary encoding. |